### DISCUSSION TITLE
Per-Seat Billing Plans feature

### DISCUSSION THREAD

**Discussion by jay2success** [bpuig/laravel-subby#123] [Q&A]
Hi,

Thanks for this awesome package.

Some applications charge users per "seat" instead of a fixed monthly price. For example, a project management application might charge $10 monthly per project such that if a user managed five projects they would be billed $50 monthly.

The way it works is there is Model connected to the subscription plan, in this case its Project model.

Each project will have its own subscription which will have separate expiration date.

I hope subby can support this since it's well known model for Saas.

Thanks again.

**Comment by bpuig** [comment]
You mean the possibility of having metered plan subscription features?

**Reply by jay2success** [reply]
Hi,

Sorry, i don't quite understand how metered plan works.

Here is simple scenario so its easier to understand.

I built SAAS which sell subscription plan based on how many sites they can build.

I have 3 plans:
1. basic: $10
2. medium: $20
3. pro: $30

But all those plans are for 1 site only.

So if they want to add more site, they must buy the same or different plan.

Example:
Jan 1st: user A buy basic plan for $10, which is for 1 site.
=> the subscription tied to Site model with name ex: aaa

Jan 10th: user A want to add more site, so they have to buy the plan again, ex: basic plan for $10
=> the subscription tied to Site model with name ex: bbb

So now the user A have 2 basic plan which totals $20 but each has different expired period since they bought it on different date.

So the subscription plan is "per seat" and can be bought multiple times.

Laravel spark has this feature but they don't support custom payment provider.

Hope this scenario clear enough.

Please tell me is it possible for this scenario, thank you.

**Reply by arimedya** [reply]
Greate feature, i hope they'll add soon. I do really need this too.
